### ISSUE TITLE
Link to Homepage in international Version - V2

### ISSUE THREAD

**Issue by anderer** [automadcms/automad-dist#1]
The website is international with DE and EN. DE is the primary route.

When on an EN site and clicking on the logo in the navbar I am routed back to the start of DE and not of EN.

I would consider this as a bug and not a feature.

**Comment by marcantondahmen** [comment]
Hi there, thanks for reporting this. It will be fixed in the next alpha release. 

Please note that this here is the distribution mirror repository and issues should be opened only in the main repository as described in the README here.

**Comment by anderer** [comment]
yes, thanks for the advice where to post - i thought i was right - hopefully next time i will get it right.
